docs/user-guide/create-page.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar-position: 3
+---
+
+# Create page
+
+This is where the actual magic happens and where you cook your resume with our
+drag-and-drop easy-to-use technology, along with multiple customizations.
```

Comment:
easy-to-use feature would be nice instead of easy-to-use technology and comma after drag-and-drop